.travis.yml

```diff
@@ -84,4 +84,4 @@ matrix:
         - package='libpng'; [[ $(brew ls --versions ${package}) ]] && { brew outdated ${package} || brew upgrade ${package}; } || brew install ${package}
 
 script:
-  - mkdir ./build && cd ./build && cmake .. ${CMAKE_BUILD_OPTIONS} && make -j2 VERBOSE=1 && make test ARGS="-j2 --output-on-failure" && make linter && make apache-rat && make docs
+  - mkdir ./build && cd ./build && cmake -DENABLE_PCAP=TRUE .. ${CMAKE_BUILD_OPTIONS} && make -j2 VERBOSE=1 && make test ARGS="-j2 --output-on-failure" && make linter && make apache-rat && make docs
```

Comment:
I think this is better off in CMAKE_BUILD_OPTIONS above

Reply:
This is going to be removed after we show it working in travis.

Reply:
I actually did not want to include it because the pcap test will require sudo to run. I've since decided it's probably harmless on a travis instance to run the pcap test since it will otherwise be disabled unless someone wants the feature.